src/test/ts/atomic/UtilsTest.ts

```diff
@@ -1,15 +1,22 @@
 import { Assertions } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
 import { isValidKey } from 'src/main/ts/Utils';
 
-UnitTest.test('UtilsTest', () => {
-
+describe('UtilsTest', () => {
   const checkValidKey = (key: string, expected: boolean) => {
     const actual = isValidKey(key);
     Assertions.assertEq('Key is valid', expected, actual);
   };
 
-  checkValidKey('onKeyUp', true);
-  checkValidKey('onkeyup', true);
-  checkValidKey('onDisable', false);
+  it('should check if key is valid onKeyUp', () => {
+    checkValidKey('onKeyUp', true);
+  });
+
+  it('should check if key is valid onkeyup', () => {
+    checkValidKey('onkeyup', true);
+  });
```

Comment:
Can group these into one test. It is worth to add a comment why testing both `onKeyUp` and `onkeyup`

Reply:
Updated in [this commit](https://github.com/tinymce/tinymce-vue/pull/434/commits/6bb5d8f1f41a7d0458ed7889335c20a06de33c71) and included comments to why we have these two test cases.